Readme Updates - Prerequisites

tonyleachsf opened the pull request:
Added Prerequisites and Sample App sections; removed reference to Sample App from 'Code Worth Looking At'

curama commented on README.md:
```diff
@@ -2,12 +2,37 @@ ios-here-sdk-dist
+
+1. A developer-enabled PayPal account ([sign up here](https://developer.paypal.com/webapps/developer/applications/myapps)).  This is the account you use to register your app.  You will receive an App ID & Secret to use with the SDK.
+2. A PayPal Here business account ([sign up here] (https://www.paypal.com/us/webapps/mobilemerchant/page/mpa/ob/geturl?onbver=2.0&amp;country.x=US&productIntentID=mobile_payment_acceptance&referringpage=ios_sdk_github&hs=login)).  This is the account that the end merchant uses, and will be the destination account of funds received. A single app can be associated with/used by one or many merchant  ...
+3. A PayPal Here swiper.  You can get one shipped to you when you create a business account in step (2), or via retailers like [Staples](http://www.staples.com/PayPal-Here-trade-Mobile-Card-Reader/product_1421621).
```
This is not absolutely necessary for a developer who wants to use our SDK right? Perhaps they can choose to use our SDK in a no-hw mode just to process payments. This bullet point should go under a Optional Field or have optional in parentheses next to it.

tonyleachsf replied:
To keep it simple, I am assuming all developers reading this will use our hardware. Bigger developers that want a different arrangement can contact us for more help with something like a noHW version. I'm hoping to not open that can of worms for a generic developer - unless we see significant demand?



On Mar 19, 2015, at 4:43 PM, curama <notifications@github.com<mailto:notifications@github.com>> wrote:


In README.md<https://github.com/PayPal-Mobile/ios-here-sdk-dist/pull/24#discussion_r26808799>:

>
>  Full class and method documentation can be [found here](http://paypal-mobile.github.com/ios-here-sdk-dist/index.html).
>
>
> +Prerequisites For Using The SDK
> +===============================
> +
> +In order to start using the PayPal Here SDK, you need the following:
> +
> +1. A developer-enabled PayPal account ([sign up here](https://developer.paypal.com/webapps/developer/applications/myapps)).  This is the account you use to register your app.  You will receive an App ID & Secret to use with the SDK.
> +2. A PayPal Here business account ([sign up here] (https://www.paypal.com/us/webapps/mobilemerchant/page/mpa/ob/geturl?onbver=2.0&amp;country.x=US&productIntentID=mobile_payment_acceptance&referringpage=ios_sdk_github&hs=login)).  This is the account that the end merchant uses, and will be the destination account of funds received. A single app can be associated with/used by one or many merchant accounts – including the developer-enabled account.  You will receive an Access Token and Refresh URL for each merchant that grants permission to your app.
> +3. A PayPal Here swiper.  You can get one shipped to you when you create a business account in step (2), or via retailers like [Staples](http://www.staples.com/PayPal-Here-trade-Mobile-Card-Reader/product_1421621).


This is not absolutely necessary for a developer who wants to use our SDK right? Perhaps they can choose to use our SDK in a no-hw mode just to process payments. This bullet point should go under a Optional Field or have optional in parentheses next to it.

—
Reply to this email directly or view it on GitHub<https://github.com/PayPal-Mobile/ios-here-sdk-dist/pull/24/files#r26808799>.

tonyleachsf commented:
I've added some more to this pull request (see my last two commits).  Let me know if I should be doing this a bit differently.

I've added a new Get Started section in Readme.md, adapted from our PDF documentation.  Please check for accuracy.

I've also moved the PayPal Access stuff to its own separate github doc .md, for reference from the main readme.  It's now commented out in readme.md.

paypaldoma commented on README.md:
````diff
@@ -2,12 +2,66 @@ ios-here-sdk-dist
+```objectivec
+	[PayPalHereSDK selectEnvironmentWithType:environment_type] 
+```
+* *environment_type* is **ePPHSDKServiceType_Sandbox** for the Sandbox environment, or **ePPHSDKServiceType_Live** for the live environment.
````
It might be worth noting that they don't have to call selectEnvironmentWithType at all.  If they don't then the SDK will default to running against Live.

tonyleachsf replied:
cool, good to know.  changed it to: 

If you want to start with test transactions (generally a good idea), you can optionally send a selectEnvironmentWithType message to PayPalHereSDK:

    [PayPalHereSDK selectEnvironmentWithType:environment_type] 

environment_type is ePPHSDKServiceType_Sandbox for the Sandbox environment, or ePPHSDKServiceType_Live for the live environment (default).
